plugins/Annotations/stylesheets/annotations.less

```diff
@@ -202,3 +225,19 @@
     border: 1px solid #e4e5e4;
     border-radius: 4px;
 }
+
+a.add-annotation {
+  margin: 0;
+}
+
+td.padding {
+  padding: 1px 5px 6px 5px;
+}
+
+td.wider {
+  width: 80px;
+}
+
+a.font   {
+  font-size: 14px;
+}
```

Comment:
Just a small side node: We should definitely try to avoid such css class names (padding, wider, font), especially when they are defined such global.